lib/features/digital_guide_view/data/readme.md

```diff
@@ -0,0 +1,14 @@
+# Requirements
+* All HTTP requests must include the authorization token (DIGITAL_GUIDE_URL)
+
+# Tips
+* To obtain the authorization token, username, or password for API login, please contact the development team
+* The complete list of endpoints is available after logging under https://przewodnik.pwr.edu.pl/swagger/
```

Comment:
make it as follows: after logging under [this ](https://przewodnik.pwr.edu.pl/swagger/) link